include/h2o/socket.h

```diff
@@ -110,6 +110,9 @@ struct st_h2o_socket_t {
     void *data;
     struct st_h2o_socket_ssl_t *ssl;
     h2o_buffer_t *input;
+    int timestamping;
+    int needs_timestamp;
+
```

Comment:
How about moving these variables right below `_skip_tracing`, with the types being changed to `unsigned : 1`?

Also, I would appreciate if we could better clarify the difference between the two flags (by changing their names, adding doc-comments, etc.), as the difference (or the necessity of having two) would not immediately be clear to people reading the code.

Reply:
sure, i added a couple comments -- if you'd like me to change the names, as well, i am happy to do so.